tests/kafkatest/tests/streams/streams_eos_test.py

```diff
@@ -67,6 +67,9 @@ def run_rebalance(self, processor1, processor2, processor3, verifier):
         self.add_streams3(processor2, processor3, processor1)
         self.stop_streams3(processor1, processor3, processor2)
         self.stop_streams2(processor1, processor3)
+
+        processor1.clean_node_enabled = True
+
```

Comment:
On line 61, this flag is set to `False` to avoid restoring the state from scratch from the changelog topic for `processor1` when `processor1` is stopped on line 66. However, when `processor1` is stopped on line 73 the state directory needs to be cleaned up because the test ends.